go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/credentials v1.10.0
 	github.com/aws/aws-sdk-go-v2/service/ecr v1.1.1
 	github.com/aws/aws-sdk-go-v2/service/sts v1.16.0
-	github.com/banzaicloud/imps/api v0.3.11
+	github.com/banzaicloud/imps/api v0.3.12
```

Comment:
With the replace in place, this doesn't need updating either.

Reply:
Yeah let me fix this in this PR to `v0.0.1` as placeholder